<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialDlg</class>
 <widget class="QDialog" name="serialDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>725</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Value Reader</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>501</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>数据信息:</string>
   </property>
   <widget class="QLineEdit" name="recvHexEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>310</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>155</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>接收信息(Hex):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="transValEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>310</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>155</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>整合数值(int):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tightTorqueEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>310</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>155</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>拧紧力矩(N.m):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tightAngleEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>210</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>155</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>拧紧角度(度):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>155</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>拧紧状态:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="flagBitEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>210</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="flagBitLabel">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>180</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>519</x>
     <y>5</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>串口设置:</string>
   </property>
   <widget class="QPushButton" name="openBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>32</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>COM端口:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>85</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="baudrateEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>82</y>
      <width>120</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="stopReadBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>停止读取</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startReadBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>开始读取</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>225</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>条形码扫描:</string>
   </property>
   <widget class="QLineEdit" name="barcodeEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>85</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>条形码1:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="barcodeEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>85</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>条形码2:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>295</y>
     <width>501</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>数据显示:</string>
   </property>
   <widget class="QLineEdit" name="groupCountEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>181</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>螺丝拧紧数目:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>121</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>( 个 / 组 )</string>
    </property>
   </widget>
   <widget class="QListWidget" name="dataListPanel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>1001</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>信息:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>145</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>工位选择:</string>
   </property>
   <widget class="QRadioButton" name="singleRadioBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>单工位</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bothRadioBtn">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>双工位</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>365</y>
     <width>491</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>数据查看:</string>
   </property>
   <widget class="QListWidget" name="productInfoPanel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="preBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>查看上一组</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>查看下一组</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="currBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>110</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>查看数据</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
